virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,43 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
-
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+|*Volumes project* list
+|Select the volume to boot your VM from.
 
-// |*Add volume*
-// |Upload a new volume or use an existing PersistentVolumeClaim (PVC).
+|*Add volume*
+|Upload a new volume or use an existing PersistentVolumeClaim (PVC).
 
-// |*Filter* field
-// |Filter boot sources by operating system.
+|*Filter* field
+|Filter boot sources by operating system.
 
-// |Search field
-// |Search boot sources by keyword.
+|Search field
+|Search boot sources by keyword.
 
-// |*Select InstanceType*
-// |Select an instance type based on the number of CPUs and amount of memory available.
+|*Select InstanceType*
+|Select an instance type based on the number of CPUs and amount of memory available.
 
-// |*VirtualMachine details*
-// |Displays the current VM settings. You can change the name of the VM during this step.
+|*VirtualMachine details*
+|Displays the current VM settings. You can change the name of the VM during this step.
 
-// |*Advanced settings*
-// |Add an authorized SSH key to the VM.
+|*Advanced settings*
+|Add an authorized SSH key to the VM.
 
-// |*Start this VirtualMachine after creation* checkbox
-// |Checked by default. Uncheck this to prevent the new VM from starting after creation.
+|*Start this VirtualMachine after creation* checkbox
+|Checked by default. Uncheck this to prevent the new VM from starting after creation.
```

Comment:
Another discovery from my UI guidelines journey -- This should be only "Clear the checkbox to prevent ..." 

Since we already know that the default is "Selected", we do not have to say what the default is because the user can see its state. If we did not know the state of the checkbox, we would have to say "Ensure that the checkbox is cleared to ...." Checkboxes take the verbs "select"/"clear".